presto-docs/src/main/sphinx/release/release-0.273.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Release 0.273
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix Disaggregated Coordinator resource group aggregation issue related to WAITING_FOR_PREREQUISITE queries.
+* Fix Iceberg $files table in case of column dropping.
+* Fix a bug where ``EXPLAIN (TYPE IO)`` did not include the index source from an index join.
+* Fix a memory overcounting issue for using varchar types that could cause queries to fail with exceeded memory limit errors.
+* Fix an error in ``tdigest_agg``.
+* Add `to_base32` and `from_base32` varbinary functions.
+* Add support for TRUNCATE TABLE statement.
+* Add support to set timeout on the query analyzer runtime to prevent long running query analysis like complex regular expressions. The timeout duration can be set by the configuration property `planner.query-analyzer-timeout` or session property `query_analyzer_timeout`.
+* Add trim_array function.
```

Comment:
Functions have to be written with :func:. Also explain what this function does.

Reply:
done:
* Add :func:`trim_array` function that can delete elements from the end of an ordinary array.